components/sp-seamless-provision/src/main/java/com/wso2telco/carbon/identity/oauth2/token/handlers/clientauth/SeamlessProvisionClientAuthHandler.java

```diff
@@ -206,5 +218,23 @@ private MobileConnectConfig getMobileConnectConfig() {
         mobileConnectConfigs = configurationService.getDataHolder().getMobileConnectConfig();
         return mobileConnectConfigs;
     }
-
+   
+	public void setPublishingDataForError(OAuthTokenReqMessageContext tokReqMsgCtx)
+					throws IdentityOAuth2Exception {
+		boolean dataPublisherEnabled = DataHolder.getInstance().getMobileConnectConfig().getDataPublisher()
+	                    .isEnabled();
+		if(dataPublisherEnabled){
+		String clientId = tokReqMsgCtx.getOauth2AccessTokenReqDTO().getClientId();
+		Map<String, String> tokenMap = new HashMap<String, String>();
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Replace the type specification in this constructor call with the diamond operator ("<>"). [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://ci.wso2telco.com:9000/coding_rules#rule_key=squid%3AS2293)